### ISSUE TITLE
AssertionError when training classifier using example

### ISSUE THREAD

**Issue by bab2155** [jakelever/kindred#7]
When working through the BioNLP Shared Task Example, I get an AssertionError when training the classifier:

```
import kindred
trainCorpus = kindred.bionlpst.load('2016-BB3-event-train')
devCorpus = kindred.bionlpst.load('2016-BB3-event-dev')
predictionCorpus = devCorpus.clone()
predictionCorpus.removeRelations()
classifier = kindred.RelationClassifier()
classifier.train(trainCorpus)
```

```
AssertionError                            Traceback (most recent call last)
<ipython-input-18-afd73a0f3df0> in <module>()
      5 predictionCorpus.removeRelations()
      6 classifier = kindred.RelationClassifier()
----> 7 classifier.train(trainCorpus)

/Users/user/.pyenv/versions/3.6.1/lib/python3.6/site-packages/kindred/RelationClassifier.py in train(self, corpus)
     83                 candidateClasses = []
     84                 for cr in candidateRelations:
---> 85                         assert len(cr.knownTypesAndArgNames) <= 1, "Kindred doesn't currently supporting training a classifier with candidate relations that are of more than one type"
     86                         if len(cr.knownTypesAndArgNames) == 0:
     87                                 candidateClasses.append(0)

AssertionError: Kindred doesn't currently supporting training a classifier with candidate relations that are of more than one type
```

**Comment by jakelever** [comment]
I'll dig into it

**Comment by jakelever** [comment]
Hey. Thanks for reporting this code. Some new code conflicted with the duplicate data in the BioNLP Shared Task set. I've fixed it (237899c97291f5d05bcec246135197669a81510a) and released a new version to PyPI (v2.2.2). Hopefully that works for you.
